modules/cnv-troubleshooting-vmware-vm-import.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+// * cnv/cnv_virtual_machines/cnv_importing_vms/cnv-importing-vmware-vm.adoc
+[id='troubleshooting-vmware-vm-import-{context}']
+= Troubleshooting a VMware virtual machine import
+
+If an imported virtual machine's status is `Import error: (VMware)`, you can check the Conversion Pod log for errors:
+
+. Obtain the Conversion Pod name:
++
+----
+$ oc get pods -n <project> | grep v2v <1>
+kubevirt-v2v-conversion-f66f7d-zqkz7            1/1     Running     0          4h49m
+----
+<1> Specify the project of your imported virtual machine.
+
+. Obtain the Conversion Pod log:
++
+----
+$ oc logs kubevirt-v2v-conversion-f66f7d-zqkz7 -f -n <project>
+----
+
+= Error messages
```

Comment:
There shouldn't be another heading level 1 (=) in the same module. I assume that this should be a level 2, but if this really should be at level 1, then it should be in a separate module file.